client/src/components/molecules/RequestsTable.tsx

```diff
@@ -78,6 +80,28 @@ const RequestsTable: FunctionComponent<Props> = (props: Props) => {
             window.location.reload();
         }
     });
+    const handleDeleteRequest = (index: number) => {
+        const req = requests[index];
+        if (req.fulfilledAt != null) {
+            // If Request is fulfilled then don't show warning
+            onDeleteRequest(index);
+        } else {
+            let canDelete = true;
+            const matchedDonations = req.matchedDonations;
+            if (matchedDonations != null) {
```

Comment:
I would use strict equality always !==

Reply:
Updated the logic checking to check if the value is falsely, which includes null and undefined.